src/helm/benchmark/static/schema.yaml

```diff
@@ -1810,6 +1810,36 @@ run_groups:
       main_name: quasi_exact_match
       main_split: test
 
+  - name: lextreme
+    display_name: LEXTREME
+    description: A Multilingual Legal Benchmark for Natural Language Understanding
+    metric_groups:
+      - accuracy
+      - calibration
+      - robustness
+      - fairness
+      - bias
```

Comment:
FYI robustness, fairness and bias are usually for scenarios that are intended to be used with the corresponding perturbations.

Reply:
Aah I see, I removed them.